src/biogeochem/DUSTMod.F90

```diff
@@ -356,86 +583,201 @@ subroutine DustEmission (bounds, &
+         wnd_frc_thr_slt_it = B_it * tmp2 / sqrt(forc_rho(c)) ! define impact threshold
+
+         ! the above formula is true for Iversen and White (1982) and Shao and Lu (2000) scheme
+         wnd_frc_thr(p) = wnd_frc_thr_slt          ! output fluid threshold
+         wnd_frc_thr_it(p) = wnd_frc_thr_slt_it    ! output impact threshold
+
+         !##############################################################################################
+         ! dmleung: here, calculate quantities relevant to the fluid threshold
+         ! standardized fluid threshold
+         wnd_frc_thr_slt_std = wnd_frc_thr_slt * sqrt(forc_rho(c) / forc_rho_std) ! standardized soil threshold friction speed (defined using fluid threshold)
+         wnd_frc_thr_std(p) = wnd_frc_thr_slt_std          ! output standardized fluid threshold
+         ! dust emission coefficient or soil erodibility coefficient (this is analogous to the soil erodibility map or prefenertial source filter in Zender; see zendersoilerodstream)
+         dst_emiss_coeff(p) = Cd0 * exp(-Ce * (wnd_frc_thr_slt_std - wnd_frc_thr_slt_std_min) / wnd_frc_thr_slt_std_min) ! save dust emission coefficient here for all grids
+
+         ! framentation exponent (dependent on fluid threshold)
+         frag_expt = (Ca * (wnd_frc_thr_slt_std - wnd_frc_thr_slt_std_min) / wnd_frc_thr_slt_std_min)  ! fragmentation exponent, defined in Kok et al. (2014a)
+         if (frag_expt > frag_expt_thr) then   ! set fragmentation exponent to be 3 or 5 at maximum, to avoid local AOD blowup
+            frag_expt = frag_expt_thr
+         end if
 
-            bd = (1._r8-watsat(c,1))*2.7e3_r8      ![kg m-3] Bulk density of dry surface soil
-            gwc_sfc = h2osoi_vol(c,1)*SHR_CONST_RHOFW/bd    ![kg kg-1] Gravimetric H2O cont
-            if (gwc_sfc > gwc_thr(c)) then
-               frc_thr_wet_fct = sqrt(1.0_r8 + 1.21_r8 * (100.0_r8*(gwc_sfc - gwc_thr(c)))**0.68_r8)
-            else
-               frc_thr_wet_fct = 1.0_r8
+         !##############################################################################################
+         !################ drag partition effect, and soil-surface friction velocity ###################
+         ! subsection on computing vegetation drag partition and hybrid drag partition factors
+         ! in Leung et al. (2023), drag partition effect is applied on the wind instead of the threshold
+         !##############################################################################################
+         ! the following comes from subr. frc_thr_rgh_fct_get
+         ! purpose: compute factor by which surface roughness increases threshold
+         !          friction velocity (currently a constant)
+
+         if (lnd_frc_mbl(p) > 0.0_r8  .AND. tlai_lu(l)<=1_r8) then
+         !if (lnd_frc_mbl(p) > 0.0_r8  .AND. ttlai(p)<=1_r8) then
```

Comment:
I'll remove this commented out section of code.

@dmleung was this here as something you tried that didn't work sufficiently well and so you removed it?

Reply:
Yes, exactly. I was using patch-level TLAI but the results were a little weird, and I realized using landunit-level TLAI gives more reasonable results. Please feel free to remove this line. 

One small thing I just noticed is that there are a few variables defined in the patch-level (e.g., `vai_Okin(p)`, `ssr(p)`) that are dependent on TLAI. Since later on I switched to the landunit-level TLAI, these variables no longer needed to be defined in patch level and could be defined in landunit-level instead (e.g., `vai_Okin(l)`, `ssr(l)`). I don't think this increases the use of computational expense much, but not sure if you want more consistency within the code. I guess this is dependent on the software engineering standard, but to me this is a very minor issue that we probably could leave out. Do you think we need to improve on this?

Reply:
Consistency is a good thing and only doing computations on the highest subgrid level that's needed is best practices as well. So I might make an issue about this. But, we won't worry about this for now.